common/hyperscaler/rules/rules_service.go

```diff
@@ -0,0 +1,27 @@
+package rules
+
+import (
+	"fmt"
+	"log"
+
+	"github.com/kyma-project/kyma-environment-broker/common/hyperscaler/rules/model"
+)
+
+type RulesService struct {
+}
+
+func NewRulesServiceFromFile(rulesFilePath string) (*RulesService, error) {
+	rulesConfig := &model.RulesConfig{}
+
+	if rulesFilePath == "" {
+		return nil, fmt.Errorf("No HAP rules file provided")
+	}
+
+	log.Printf("Parsing rules from file: %s\n", rulesFilePath)
+	rulesConfig, err := rulesConfig.Load(rulesFilePath)
```

Comment:
Is this a local value? `rulesConfig`

Reply:
Yes, it will be that way. Subsequent PRs do parsing on the rulesConfig putting the results in RulesService.

Reply:
I was just puzzled because this value is lost in current implementation.
It is not returned. It is not passed further.